test/e2e/example.test.ts

```diff
@@ -0,0 +1,6 @@
+import { test, expect } from '@playwright/test';
+
+test('example test', async ({ page }) => {
+   await page.goto('http://localhost:5173');
+   await page.getByRole('button', { name: 'Log In' }).click();
+});
```

Comment:
You checked that it is working,great.
Now lets delete it as it is not relevant :)